src/pages/ProductList/ProductList.js

```diff
@@ -1,54 +1,75 @@
 import React, { useState, useEffect } from 'react';
-import { Link, Outlet } from 'react-router-dom';
-import Nav from './../../components/Nav/Nav';
-import Category from './Category/Category';
+import { useLocation, useNavigate } from 'react-router-dom';
+import ProductListLayout from './ProductListLayout/ProductListLayout';
 import FilterOpen from './FilterOpen/FilterOpen';
-import BaseInfo from '../../components/BaseInfo/BaseInfo';
-import Footer from './../../components/Footer/Footer';
+import Category from './Category/Category';
+import CategoryList from './CategoryList/CategoryList';
 import './ProductList.scss';
+import API from '../../config/config';
 
 const ProductList = () => {
+  const { allProducts, category } = API;
+  const location = useLocation();
+  const navigate = useNavigate();
   const [productList, setProductList] = useState([]);
+  const [categoryInfo, setCategoryInfo] = useState([]);
   const [isFilterOpen, setIsFilterOpen] = useState(false);
 
   useEffect(() => {
-    fetch('http://localhost:3000/data/category_products.json')
+    fetch(`${allProducts}/${location.search}`)
+      .then(res => res.json())
+      .then(data => setProductList(data.result));
+  }, [location.search]);
```

Comment:
```suggestion
    fetch(API.allProducts`${location.search}`)
      .then(res => res.json())
      .then(data => setProductList(data.result));
  }, [location.search]);
```

이렇게 해보세요
`/` 가 없는 이유는 config 리뷰 보시면 거기에도 코멘트 달아놨습니다

Reply:
config에 반영하고 위와 같이 수정하였습니다!